### DISCUSSION TITLE
Should the `Position` enum be moved to `ratatui::layout`?

### DISCUSSION THREAD

**Discussion by kdheepak** [ratatui/ratatui#1113] [Discussion]
Currently `Alignment` enum comes from `ratatui::layout::alignment`, e.g. `ratatui::layout::alignment::Alignment::Right` but `Position` enum comes from `ratatui::widgets::block::title`, e.g. `ratatui::widgets::block::title::Position::Bottom`.

Here's an example usage:

```rust
ratatui::widgets::block::Title::from("Title")
    .alignment(ratatui::layout::alignment::Alignment::Right)
    .position(ratatui::widgets::block::title::Position::Bottom)
```

**Comment by kdheepak** [comment]
I guess there's another `Position` that's `Position { x, y }` in `rect`.

Maybe `title`'s `Position` should be renamed to `TitlePosition`?

**Reply by janTatesa** [reply]
I did that in my deprecate title pr. However it waits a lot for getting merged

**Reply by joshka** [reply]
yep :)

**Comment by janTatesa** [comment]
The title::Position enum is going to be deprecated in my pr. Instead, it's moved to block::TitlePosition because the title module is deprecated entirely

**Reply by kdheepak** [reply]
Thanks for making the PR! I didn't get a chance to look at that yet, I'll close this discussion!
